spec/features/manage_cmr/subscriptions/index_subscriptions_spec.rb

```diff
@@ -17,54 +30,73 @@
         expect(page).to have_css('h2.current', text: 'Manage CMR')
         expect(page).to have_content('View Subscriptions')
       end
+    end
 
-      # The two tests in this context need to be revised when we have real endpoints
-      context 'when viewing the index page' do
-        before do
-          visit subscriptions_path
-        end
+    context 'when viewing the index page' do
+      before do
+        visit subscriptions_path
+      end
+
+      # These tests can be improved when we can reset_provider. They should have
+      # only 2 subscriptions on them, but if the subscription tests are run
+      # together, CMR does not always finish deleting subscriptions from other
+      # tests before starting this one, so it fails at the commented out line.
+      it 'displays sample subscriptions' do
```

Comment:
1. I don't know if 'sample' is the right descriptor
2. I think you want to be more explicit here that it is showing the subscriptions but not the links for which they don't have access